pip/vcs/__init__.py

```diff
@@ -327,7 +327,9 @@ def run_command(self, cmd, show_stdout=True, cwd=None,
             # errno.ENOENT = no such file or directory
             # In other words, the VCS executable isn't available
             if e.errno == errno.ENOENT:
-                raise BadCommand('Cannot find command %r' % self.name)
+                raise BadCommand(
+                    'Cannot find command %r - do you have '
+                    '%r installed and in your PATH?' % self.name)
             else:
```

Comment:
You have 2 instances of %r but only one argument. Either add a second copy of ```self.name``` to the RHS or just go with "... do you have it installed ...".